### PR TITLE
Minor change on curation queue style; Make sendEmail return promise;

### PR DESCRIPTION
**Pull request by zhx828**

### CODE REVIEW

File: web/yo/app/scripts/directives/curationQueue.js [hunk 1]
```diff
@@ -237,38 +268,55 @@ angular.module('oncokbApp')
...
+                        .then(function() {
+                            deferred.resolve();
+                        }, function(error) {
+                            deferred.reject(error);
+                        });
+                    return deferred.promise;
                 }
+
                 $scope.markEmailSent = function() {
                     $scope.emailSent = false;
```
Comment (JiaoJiao123) [inline web/yo/app/scripts/directives/curationQueue.js]: change to the email object you added.

File: web/yo/app/views/curationQueue.html [hunk 2]
```diff
@@ -100,12 +113,17 @@ <h3 class="headerStyle">Curation Queue
...
+                <button tyle="button" class="btn btn-success"
+                        ng-disabled="!curatorsToNotify || curatorsToNotify.length === 0 || emailSent"
+                        ng-click="sendEmail(curatorsToNotify)">
+                    <span ng-show="!email.status.sending">Notify</span>
+                    <i ng-show="email.status.sending" class="fa fa-spinner fa-spin"></i>
+                </button>
             </div>
             <div class="col-sm-2">
-                <p class="message">{{emailReturnMessage}}</p>
+                <span ng-if="email.returnMessage">{{email.returnMessage}}</span>
```
Comment (JiaoJiao123) [inline web/yo/app/views/curationQueue.html]: could you put back the 'message' class?

File: web/yo/app/views/curationQueue.html [hunk 3]
```diff
@@ -100,12 +113,17 @@ <h3 class="headerStyle">Curation Queue
                 </select>
             </div>
             <div class="col-sm-2">
-                <button tyle="button" class="btn btn-success" ng-disabled="!curatorsToNotify || curatorsToNotify.length === 0 || emailSent" ng-click="sendEmail(curatorsToNotify)">Notify</button>
+                <button tyle="button" class="btn btn-success"
+                        ng-disabled="!curatorsToNotify || curatorsToNotify.length === 0 || emailSent"
```
Comment (JiaoJiao123) [inline web/yo/app/views/curationQueue.html]: change emailSent to the email object

File: web/yo/app/scripts/directives/curationQueue.js (hunk 1 above)
Comment (zhx828) [reply web/yo/app/scripts/directives/curationQueue.js]: Removed emailSent

File: web/yo/app/views/curationQueue.html (hunk 2 above)
Comment (zhx828) [reply web/yo/app/views/curationQueue.html]: Removed message class from css file. Use `<b></b>` instead

File: web/yo/app/views/curationQueue.html (hunk 3 above)
Comment (zhx828) [reply web/yo/app/views/curationQueue.html]: Use `email.status.sending` instead

**Pull request merged**